[PATCH] Fix a compile error 

Upstream-Status: Pending

Fix a compile error by conditional using 'catal' since catal
is only defined when LIBXML_CATALOG_ENABLED is enabled.

Signed-off-by: Roy Li <rongqing.li@windriver.com>
---
 LibXML.xs |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/LibXML.xs b/LibXML.xs
index 66da04b..45da681 100644
--- a/LibXML.xs
+++ b/LibXML.xs
@@ -2777,9 +2777,11 @@ _default_catalog( self, catalog )
         xmlCatalogPtr catal = INT2PTR(xmlCatalogPtr,SvIV(SvRV(catalog)));
 #endif
     INIT:
+#ifdef LIBXML_CATALOG_ENABLED
         if ( catal == NULL ) {
             croak( "empty catalog\n" );
         }
+#endif
     CODE:
         warn( "this feature is not implemented" );
         RETVAL = 0;
-- 
1.7.10.4

